lib/releaselib.sh

```diff
@@ -219,15 +223,10 @@ release::set_build_version () {
     # *specifically* for a release branch with a version in it (non-master).
     if [[ "$branch" =~ release-([0-9]{1,})\. ]]; then
       if [[ $build_sha1 != $branch_head ]]; then
-        # TODO: Figure out how to curl a list of last N commits
-        #       So we can return a message about how far ahead the top of the
-        #       release branch is from the last good commit.
-        #commit_count=$(git rev-list $build_sha1..${branch_head} |wc -l)
-        commit_count=some
+        commit_count=$((first_build_number-build_number))
         logecho
         logecho "$ATTENTION: The $branch branch HEAD is ahead of the last" \
-                "good Jenkins run by $commit_count commits." \
-                "Wait for Jenkins to catch up."
+                "good run by $commit_count commits."
```

Comment:
Probably more accurate to say "by $build_count builds"? We don't necessarily create a build for every commit.

Reply:
Understood that the CI can skip commits, but this is counting the actual number of commits.

Reply:
Oh how does that work?

Reply:
Ohh I see. It's coming from the tag calculated by git. Clever! :)

Reply:
Yeah I realized it by accident when looking at something else.  I thought that TODO would be there forever. :)